apps/web/src/views/Mev/hooks/index.ts

```diff
@@ -1,11 +1,13 @@
 import { ChainId } from '@pancakeswap/chains'
 import { useQuery } from '@tanstack/react-query'
 import useActiveWeb3React from 'hooks/useActiveWeb3React'
-import { useCallback, useMemo } from 'react'
+import { useCallback } from 'react'
+import { Address } from 'viem'
 import { bsc } from 'viem/chains'
+import { useWalletClient } from 'wagmi'
 
-async function fetchMEVStatus(): Promise<boolean> {
-  if (!window.ethereum || (!window.ethereum as any)?.request) {
+async function fetchMEVStatus(account: Address): Promise<{ mevEnabled: boolean; isError: boolean }> {
+  if (!window.ethereum || (!window.ethereum as any)?.request || account === '0x') {
```

Comment:
we should use wallet client here as well?

Reply:
fixed